src/test/signatures.blockchain.test.js

```diff
@@ -1,46 +1,113 @@
 // @flow
 import { List } from 'immutable'
-import { buildSignaturesFrom } from '~/logic/safe/safeTxSigner'
+import { generateSignaturesFromTxConfirmations } from '~/logic/safe/safeTxSigner'
+
+const makeMockConfirmation = (address: string) => ({ owner: { address } })
 
 describe('Signatures Blockchain Test', () => {
   it('generates signatures in natural order even checksumed', async () => {
     // GIVEN
     const userA = 'baR'
     const userB = 'baz'
     const userC = 'foZa'
-    const sender = 'foZ'
+    const userD = 'foZ'
```

Comment:
I would probably have taken valid hex chars (so 0-9a-f and not O,Z and R ... but doesn't really matter :) )